### ISSUE TITLE
Need License

### ISSUE THREAD

**Issue by joverbey** [orlandoacevedo/MCGPU#27]
We should add a license to make sure we're clear about our terms of modification, redistribution, etc.

TL;DR Legal does a nice job of summarizing the various licenses: https://tldrlegal.com/

A 3-clause BSD license [https://tldrlegal.com/license/bsd-3-clause-license-(revised)] is probably the most common liberal license, which places almost no restrictions on what people can do with the code.  For example, someone could create a derived product, build a closed-source commercial product based on the MCGPU code, etc.

The Apache license [https://tldrlegal.com/license/apache-license-2.0-(apache-2.0)] is similar.

The GPL [https://tldrlegal.com/license/gnu-general-public-license-v2] allows derivatives but prevents use in closed-source products.  The problem is that it's a "viral" license: if you copy any GPL-licensed code (even a few lines) into another application, that application must be open sourced, and it must also be licensed under the GPL.

I would lean toward a 3-clause BSD or Apache license, but @orlandoacevedo should make this call...

**Comment by mrhwick** [comment]
There is a license at the bottom of the `README.md` that states a GPL for this project. Maybe worth copying it from http://www.gnu.org/licenses/gpl.txt into a `LICENSE` file.
![image](https://cloud.githubusercontent.com/assets/865759/15397021/991de50c-1dac-11e6-958d-57c33174f426.png)

**Comment by orlandoacevedo** [comment]
Thanks Matt! The GPL license was chosen at the request of NSF when the project was funded.

**Comment by joverbey** [comment]
Funny I edited the Readme and didn't even see the license at the end of it.  Sorry.  Anyway, thanks for adding the LICENSE file.
